pkgs/mime/tool/mime_types.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:mime/mime.dart';
+import 'package:mime/src/default_extension_map.dart';
+
+void main(List<String> args) {
+  // Get all the mime types.
+  final mimeTypes = defaultExtensionMap.values.toSet().toList()..sort();
+
+  // Find all their extensions.
+  final mimeToExts = <String, List<String>>{};
+
+  for (final entry in defaultExtensionMap.entries) {
+    final ext = entry.key;
+    final mime = entry.value;
+
+    mimeToExts.putIfAbsent(mime, () => []).add(ext);
+  }
+
+  // Emit the table.
+  const marker = '<!-- table -->\n';
+  final file = File('doc/mime_types.md');
+  var contents = file.readAsStringSync();
+  final prefix =
+      contents.substring(0, contents.indexOf(marker) + marker.length);
+  final suffix = contents.substring(contents.lastIndexOf(marker));
+
+  final buf = StringBuffer();
+  buf.write('''
+| MIME type | Default ext | Additional exts |
+| - | - | - |
+''');
+
+  for (final mime in mimeTypes) {
+    final defaultExt = extensionFromMime(mime)!;
+    final exts = mimeToExts[mime]!;
+
+    exts.remove(defaultExt);
+    exts.sort();
+
+    final decribeMime = '`$mime`';
+    final decribeExt = '`$defaultExt`';
+    final describeExts = exts.map((ext) => '`$ext`').join(', ');
+
+    buf.writeln('| $decribeMime | $decribeExt | $describeExts |');
```

Comment:
Just to be a little weird about formatting, consider putting the spaces around the `describeExts` into their value, and only if it's not empty:
```dart
    final additional = exts.isEmpty ? '' : ' `${exts.join('`, `')}` ';
    // ...
    buffer.writeln('| `$mime` | `$ext` |$additional|');
```
That way something with no extra extensions won't have two spurious spaces between in the `|  |`. (Which have no effect on Markdown parsing, but it irks me.)

Reply:
I'd played around with emitting the columns w/ some min width - so that things would line up visually even when viewed as text. That still looked pretty ragged - many of the mime names are long, so no reasonable column width made things line up.